onyx-remote-driver/src/com/onyx/persistence/manager/impl/DefaultSocketPersistenceManager.java

```diff
@@ -89,7 +87,7 @@ public void saveEntities(List<? extends IManagedEntity> entities) throws EntityE
         try {
             proxyPersistenceManager.saveEntities(entities);
         } catch (RemoteException e) {
-            throw new EntityException(e);
```

Comment:
No I don't mean use the constructor, I mean create an overloaded constructor instead